src/models/søknad.ts

```diff
@@ -31,3 +32,29 @@ export interface IPeriode {
   fra: Date;
   til: Date;
 }
+
+export interface ISpørsmålOgMultiSvar {
+  nøkkel: string;
+  spørsmål_tekst: string;
+  svar_tekst: string;
+}
+
+export interface ISpørsmålOgJaNeiSvar {
+  nøkkel: string;
+  spørsmål_tekst: string;
+  svar: boolean;
+}
+
```

Comment:
Er det kanskje bedre at` ISpørsmålOgMultiSvar `og `ISpørsmålOgJaNeiSvar` merges til:

```
export interface ISpørsmålOgSvar {
  nøkkel: string;
  label: string; // tidligere spørsmål_tekst
  verdi: boolean | string; // tidligere svar
}
```

Reply:
Tenker også det er greit å holde de separat for å unngå sjekk av type hver gang. _Hvis_ vi skal gjøre noe for å unngå å ha flere varianter av denne, så tenker jeg det er bedre å bare bruke `ISpørsmålOgMultiSvar` (altså at vi dropper boolean og bare kjører string for "Ja" og "Nei"), men greit å bruke begge.

Reply:
oki, da forblir den som det er satt opp!